lgn-provers/src/params/mod.rs

```diff
@@ -340,46 +254,52 @@ impl ParamsLoader
                 },
             )
             .collect();
```

Comment:
I think it makes sense to remove these collects here, I think it is just translating the `{"": "<hash>"}` to `{"<filename>": "<hash>"}`, I suppose the checks can be done more directly by just looking at the map of expected and testing against this. This is more for a follow up though.